app/kubernetes-ingress-controller/1.3.x/index.md

```diff
@@ -46,12 +46,12 @@ load configuration intended for other instances or other ingress controllers.
 
 ### Security
 
-Please refer to [this document](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/security) to understand the
+Please refer to the [Security concepts guide](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/security) to understand the
 default security settings and how to further secure the Ingress Controller.
 
 ## Guides and Tutorials
 
-Please browse through [guides][guides] to get started or understand how to configure
+Browse through [guides][guides] to get started or understand how to configure
```

Comment:
```suggestion
Browse through the [how-to guides][guides] to get started or understand how to configure
```
Not sure if that's the best term, but I think this needs a qualifier. On its own, "the guides" sounds vague.